client/app/containers/Blog/index.js

```diff
@@ -55,8 +56,11 @@ export class Blog extends Component {
 
   render() {
     const {
+      currentPage,
       focusedPost,
       loading,
+      maxPages,
+      onGetPosts,
       posts,
       routeParams,
     } = this.props;
```

Comment:
actually... it makes sense you can't since the this is scoped to the class not render... derp on my end.